runner.py

```diff
@@ -915,6 +937,27 @@ def store_phases(self):
             WHERE id = %s
             """, params=(json.dumps(self.__phases), self._project_id))
 
+    def read_stdout_logs(self):
+        print(TerminalColors.HEADER, '\nSaving container logs', TerminalColors.ENDC)
+        for container_name in self.__containers.values():
+            log = subprocess.run(
+                ['docker', 'logs', '-t', container_name],
+                check=True,
+                encoding='UTF-8',
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+            )
+            if log.stdout:
+                self.add_to_log(container_name, log.stdout)
+
+    def save_stdout_logs(self):
+        logs_as_str = '\n\n'.join([f"{k}:{v}" for k,v in self.__stdout_logs.items()])
```

Comment:
Shouldn't you escape the data here. Otherwise you could log something like `<script> ...`  and this will just be outputted?